src/components/App.js

```diff
@@ -38,11 +38,12 @@ class App extends Component {
     if ('geolocation' in navigator && !infoInLocalStorage) {
       getLocation().then(response => {
         const { latitude, longitude } = response.coords;
-        sendLocation({ longitude, latitude });
+        sendLocation({ longitude, latitude }).then(() => {
```

Comment:
I think you can do `sendLocation({longitude, latitude}).then(this.getWeatherInformation)` since it doesnt get any parameters?